example/area_graph.html

```diff
@@ -0,0 +1,57 @@
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Nugget.js Graphing Library</title>
+    <script src="/dependencies/d3.js"></script>
+    <script src="js/require.min.js"></script>
+    <link type="text/css" href="/css/nugget.css" rel="stylesheet">
+    <link type="text/css" href="example.css" rel="stylesheet">
+</head>
+<body>
+    <h1>Nugget</h1>
+    <h4>Area Graph Example</h4>
+    <div class="svg_container">
+        <svg id="area_graph_nugget"></svg>
+    </div>
+    <script>
+        require.config({
+            // Development
+            baseUrl: '/build'
+
+            // Production
+            // baseUrl: '/release/'
+        });
+    </script>
+
+    <script>
```

Comment:
Nice, thanks for adding an example :+1: I've been slowly consolidating this stuff on a branch that I need to revisit at some point. It will be much easier to add these (and with less copypasta) in the future.